Gemfile

```diff
@@ -25,3 +25,4 @@ gem "concurrent-ruby", "~> 1.1"
 
 # for releases.rss.ruby
 gem 'rss'
+gem 'faraday-follow_redirects'
```

Comment:
at the very least you need to run bundle install, then commit the new updated gemfile.lock